### ISSUE TITLE
App closes after picking an emoji

### ISSUE THREAD

**Issue by SjoerdB93** [mijorus/smile#24]
I'm suspecting it might be by design, so I'm not really sure if it's a bug, but if I click an emoji, it copies it nicely to my clipboard but it also closes the application immediately. Personally I would prefer if it would keep the application open so that I can then directly find and copy another emoji. 

If this is by design, possibly it would be a good alternative to make it a preference option? Just a switch/checkbox in the preferences menu?

Video related 😃
[Screencast from 2023-01-10 12-58-17.webm](https://user-images.githubusercontent.com/68477016/211545927-4b080d18-9a76-4507-a790-b9a6876e792f.webm)

I'm running Fedora Silverblue 37 by the way, on the latest GNOME 43.2 build.

Edit: I also notice it keeps running in the background. As Smile is written in GTK, I expect it may be expecting to run within the GNOME environment, which by default doesn't have any app indicators. So on e.g. Fedora there's no way of telling it's still running in the background. 
I'm not trying to push GNOME HIG design guides on external applications, but just wanted to throw that out there.

**Comment by mijorus** [comment]
Hi @SjoerdB93 ,
first of all I just want to point out that a gtk4 is in the making, so I am currently busy on that and not planning any new feature release untile it's done, except bug fixes.

Let's now address your points:

- Yes, it closes by design, but there is a trick for multi-selecton: if you highlight with the keyboard and press `shift + enter` while Smile will select more than one emoji, until you press `enter` (which selects one more emoji) or `ctrl+enter` (closes immediately without selecting any other emojis). You will see a preview the emojis you are selecting on the top.
- The reason why it run in the background is because Smile doesn't actually close, but rather hides itself: this, in testings, cuts up to 200ms. I am aware that in newer versions on GNOME this behaviour is causing some problems, and I'm working to fix them.
- there isn't any appindicator

**Comment by SjoerdB93** [comment]
Hi,

Thanks for the elaborate reply. To be honest, I was actually unaware that it was not running GTK4. I'm using the adw-gtk3 theme on my system, and it does a pretty good job of disguising GTK3 apps as if they were made with libadwaita. I absolutely agree that it's best to get that done first before working on any new features.

Regarding the issue here. It was mainly just that I was playing around with the application. I wanted to just select an emoji, copy this externally. Select another one, copy that externally etc.. and I had to keep restarting smile. If I want to use multiple different emojis though, I guess multi-selection works as well. I was honestly not aware that this was an option, should have read the readme. Perhaps having ctrl+mouse click would be a nice alternative also for the multi-selection option. As fully switching to keyboard while doing main operations with the mouse is a bit counter-intuitive.

Not sure if I'm the biggest fan of keeping it running in the background to save less than a second in launch time. It's not a lot of resources, but it's still there just taking 60 Mb of RAM when I'm not using it. But I guess that's a design decision. Maybe in the future it would be a nice to have that as on option.

Not that I'm some hard-core Smile poweruser. I just happened to stumble upon this on Flathub, and played around with it. Just wanted to give my general impression, not trying to say that my impression is the objective right one :)
